pkg/providers/pricing/controller.go

```diff
@@ -49,10 +53,18 @@ func (c *Controller) Builder(_ context.Context, m manager.Manager) corecontrolle
 }
 
 func (c *Controller) updatePricing(ctx context.Context) error {
+	logging.FromContext(ctx).Infof("Updating spot price information")
```

Comment:
We have a log right now that's directly in the pricing provider for [spot](https://github.com/aws/karpenter-provider-aws/blob/main/pkg/providers/pricing/pricing.go#L367) and [on-demand](https://github.com/aws/karpenter-provider-aws/blob/main/pkg/providers/pricing/pricing.go#L207) pricing that indicates when we have made updates to our pricing information. I don't think that we also need info logs that indicate that we are starting the update too

Reply:
removed this